website/docs/configuration/modules.html.md

```diff
@@ -423,10 +425,54 @@ deleted.
 
 ### Limitations when using module expansion
 
-Modules using `count` or `for_each` cannot pass different sets of providers to different instances.
+Modules using `count` or `for_each` cannot include configured `provider` blocks within the module.
+Only [proxy configuration blocks](#proxy-configuration-blocks) are allowed.
+
+If a module contains allowed provider blocks, the calling module block must be have the
```

Comment:
When I first read this, I thought "allowed provider block" was a concept and it sidetracked me a bit. Is this sentence referring only to proxy configuration blocks?

Reply:
Yes it is, those mentioned in the previous sentence ... hmm. Some shaping is needed here, I think. Will try.